engine/src/main/java/com/sandboni/core/engine/sta/graph/LinkType.java

```diff
@@ -54,6 +54,8 @@
     // invoke my pattern matching
     HTTP_MAP("HTTP Pattern", false),
 
+    HTTP_MAP_SELONI("HTTP Pattern Seloni", false),
```

Comment:
What would be the difference with HTTP_MAP?

Reply:
Only for future debugging - so we will have an indication if the mapping was created due to seloni or not

Reply:
for a future debug, if we would like to know if it was taken from Seloni or not